### ISSUE TITLE
Request: multiple entities identify state

### ISSUE THREAD

**Issue by biau** [kalkih/mini-graph-card#72]
Hi,
I would like to use this card to display tree different rooms temperatures.
If I use show_state: true, I can see the first entity state on the left  and the second and third entities on the right but with no label in front so I have no way to know the room the temperature is from.
Maybe you could use the same color for the font as for the lines ?

**Comment by kalkih** [comment]
Hey,
I've thought about adding the corresponding line color next to the state, similar to the legend.

**Comment by biau** [comment]
That would be nice!

**Comment by kalkih** [comment]
New release available [v0.3.1](https://github.com/kalkih/mini-graph-card/releases/tag/v0.3.1)

You can now add a color indicator next to the state with the entity option `show_indicator`.
Or have the state font colored in the entity color through the entity option `state_adaptive_color`.

```yaml
#example
- type: custom:mini-graph-card
  entities:
    - entity: sensor.example1
    - entity: sensor.example2
      show_state: true
      show_indicator: true
    - entity: sensor.example3
      show_state: true
      state_adaptive_color: true
```

Let me know if you have any questions.

**Comment by biau** [comment]
Excellent thanks!
state_adaptive_color is really great and show_indicator too but does not work for the first entity.
